tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -361,12 +361,7 @@ bool IsDeviceCompatible(const RemapperContext& ctx, Pattern& matched) {
 }
 
 bool IsSupportedActivation(const NodeDef& node) {
-// Disable LeakyRelu temporarily before MKL PR is merged.
-#ifndef INTEL_MKL
   return IsRelu(node) || IsRelu6(node) || IsElu(node) || IsLeakyRelu(node);
```

Comment:
(No action required until #42173 is merged.)
This (and probably some other parts of this PR) will have merge conflict with #42173. Sorry I couldn't get to #42173 sooner!

Reply:
Yeah, I will fix the conflicts after https://github.com/tensorflow/tensorflow/pull/42173 is merged